components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/DevPortalConstants.java

```diff
@@ -0,0 +1,14 @@
+package org.wso2.carbon.apimgt.impl.dao.constants;
```

Comment:
License header? Please check all your files and add this change.

Reply:
Sorry for missing this. Added the required change by [4th set of required changes](https://github.com/wso2/carbon-apimgt/pull/12874/commits/2a759c6831d1b582c753f7f87a20fde45f3232ad). Checked all the files. Mow license headers are there for all the new files except auto generated ones. Skipped auto generated ones since they will get override in the build.